src/main/java/org/opentripplanner/routing/api/request/RoutingRequest.java

```diff
@@ -168,7 +171,7 @@
      * The epoch date/time in seconds that the trip should depart (or arrive, for requests where
      * arriveBy is true)
      */
-    private long dateTime = new Date().getTime() / 1000;
+    private Instant dateTime = Instant.now();
```

Comment:
Beautiful!

Reply:
Yes, the Instant is a wrapper around `epocSeconds`, and getting rid of all the slightly different accessors for `dataTime` felt good. I see now that I forgot to update that JavaDoc, but I think I will draw the line here - there are 95% technical improvements in this PR...